src/tools/perf/api/libperf.h

```diff
@@ -164,12 +172,20 @@ typedef struct ucx_perf_rte {
     ucx_perf_rte_recv_func_t         recv;
     ucx_perf_rte_exchange_vec_func_t exchange_vec;
 
-    /* Handle results */
-    ucx_perf_rte_report_func_t       report;
```

Comment:
shall we remove ucx_perf_rte_report_func_t definition?

Reply:
fixed